bridges/TwitterBridge.php

```diff
@@ -210,6 +210,16 @@ public function getURI()
         }
     }
 
+    public function getFullText($id)
+    {
+        $url = sprintf(
+            'https://cdn.syndication.twimg.com/tweet-result?id=%s&lang=en',
+            $id
+        );
+
+        return json_decode(getContents($url), false);
```

Comment:
Is this a good idea? Another failure point

Reply:
I try to get full-text tweet from it since tweet returned from search endpoint if it too long, it get truncated.